packages/components/src/List/DisplayTable/DisplayTable.component.js

```diff
@@ -147,7 +149,14 @@ function ListHeader(props) {
 	return (
 		<tr>
 			{(isSelected && onToggleAll) && (<th />)}
-			{columns.map((column, index) => (<th key={index}>{headerContent(column, sort)}</th>))}
+			{columns.map((column, index) => (
+				<th key={index}>
+					{column.label}
+					<div aria-hidden="true" className={theme.header}>
+						{headerContent(column, sort)}
```

Comment:
I have issues passing some native props to the Button via Action.
Instead of going inot this dev in this PR, here what I propose : 
* if everything else is ok, approve this PR and let's merge it to be in the next release
* I continue to work on that in another PR right now

What do you think ?